std/internal/math/biguintcore.d

```diff
@@ -1288,7 +1289,7 @@ BigDigit [] add(const BigDigit [] a, const BigDigit [] b) pure nothrow
 
 /**  return x + y
  */
-BigDigit [] addInt(const BigDigit[] x, ulong y) pure nothrow
+BigDigit [] addInt(const BigDigit[] x, ulong y) pure nothrow @safe
```

Comment:
Be careful about bounds check in '@safe' code, it may heavily degrade performance.